CS157A-Team18/CS157A-Team18
# [D-16] Write Query for Signup Function

Changes:
- Installed `aes256` dependency for password encryption
- Created query for signup function

To test:
1. Pull this branch using the instructions here: https://stackoverflow.com/questions/9537392/git-fetch-remote-branch
2. Replace the `.env` file with the updated version from Google Drive
3. In the **root** directory, do `npm install`
4. In the **root** directory, do `PORT=3001 npm start`
5. Navigate to `http://localhost:3001/`. You should see the following response:
![image](https://user-images.githubusercontent.com/36905285/67903946-29ccda00-fb2a-11e9-8e7e-156628e15ed8.png)

- The first test case attempts to add a user that already exists in the DB. The program should return an error (which is displayed in the actual output). 
- The second test case adds a random username to the DB. Connect to the Amazon RDS DB and verify that the username displayed in the browser is added.
